resample_spatial.json

```diff
@@ -110,7 +110,7 @@
         }
     ],
     "returns": {
-        "description": "A raster data cube with values warped onto the new projection.",
+        "description": "A raster data cube with values warped onto the new projection. It has the same dimensions and the same dimension properties (name, type, labels, reference system and resolution) for all non-spatial or vertical spatial dimensions. For the horizontal spatial dimensions the name and type have not changed, but the reference system has changed and the labels and resolution may have changed.",
```

Comment:
> For the horizontal spatial dimensions the name and type have not changed, but the reference system has changed and the labels and resolution may have changed.

For the horizontal spatial dimensions the name and type remain unchanged, but the reference system changes and the labels and resolution may change.

Reply:
The reference system does not have to change if one only want to resample the data, which is allowed by the process. I propose:
"For the horizontal spatial dimensions the name and type remain unchanged, but reference system, labels and resolution may change."

Reply:
Good point, @lforesta.

Will extend it a bit to "For the horizontal spatial dimensions the name and type remain unchanged, but reference system, labels and resolution may change depending on the given parameters."